auth_totp/models/res_users_authenticator.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+from openerp import api, fields, models
+
+_logger = logging.getLogger(__name__)
+try:
```

Comment:
Why? There's no carriage return in the example given here: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports.